assets/src/edit-story/components/library/panes/media/media3p/categoryPill.js

```diff
@@ -56,10 +57,24 @@ PillContainer.propTypes = {
   isSelected: PropTypes.bool,
 };
 
-const CategoryPill = ({ index, title, isSelected, onClick }) => {
+const CategoryPill = ({
+  index,
+  title,
+  isSelected,
+  isExpanded,
+  setIsExpanded,
+  onClick,
+}) => {
   const ref = useRef();
 
-  useRovingTabIndex({ ref, isRowBasedGallery: true });
+  useRovingTabIndex({ ref, isRowBasedGallery: true }, [isExpanded]);
+
+  useKeyDownEffect(
+    ref,
+    !isExpanded ? 'down' : [],
```

Comment:
Can you add a comment explaining why we have different keys depending on isExpanded (for posterity)?

Reply:
Done.